checkout/module.go

```diff
@@ -109,15 +104,14 @@ func (m *Module) CueConfig() string {
 	// language=cue
 	return `
 commerce: checkout: {
-	Redis :: {
+	Redis : {
```

Comment:
this seems to be wrong, Redis here is a template that can be reused in other places. You now changed it to a concrete property.